scripts/ci/diff.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# be stricter
+set -eou pipefail
+
+# Run the public API diff tool from uniffi-zcash-cli
+#
+# Takes the following function args:
+# $1 - outdated librustzcash dependency where the version is not latest, in format - "crate_name;..."
+# $2 - uniffi-zcash-lib Cargo.toml path
+# $3 - absolute path of the directory, that's passed to the diff tool
+#
+# Exports:
+# ${lib_name}_colored.diff file for every outdated library - for output in the github workflow console
+# ${lib_name}.diff file for every outdated library - for output in the issue
+
+diff() {
+	local outdated_libs=$1
+	local uniffi_cargo_path=$2
+	local grep_dir=$3
+	if [[ -z "$outdated_libs" || -z "$uniffi_cargo_path" || -z "$grep_dir" ]]; then
+		echo "required parameter for diff() is empty" 1>&2
+		exit 1
+	fi
+
+	IFS=';' read -ra arr <<<"$outdated_libs"
+	for lib_name in "${arr[@]}"; do
```

Comment:
you could probably replace all this reading to array and then iterating it with
```suggestion
	echo "$outdated_libs" |
	IFS=';' while read -r lib_name; do
```
maybe even without IFS as per other comment